### ISSUE TITLE
a11y - Mat-tabs markup needs updated to include aria-expanded attribute to be compliant with role="tab" 

### ISSUE THREAD

**Issue by VamsiDharmaji** [angular/material2#11388]
#### Bug, feature request, or proposal:
Matertial tab should valid markup and follow the rules of the ARIA specification for Role = tab.

#### What is the expected behavior?
aria-expanded attribute must be toggled between "true" or "false" upon each role="tab" element to reflect the expanded state of each Tab. 

**Aria - Specification  :**
If aria-multiselectable is null or set to "false" within Tablist, and focus movement does not cause each Tab to automatically expand, aria-selected="true" should follow focus regardless which Tab is expanded while all other Tabs should include aria-selected="false", and the aria-expanded attribute must be toggled between "true" or "false" upon each role="tab" element to reflect the expanded state of each Tab. 

Review the ARIA Role Conformance Matrices at: http://whatsock.com/training/matrices/#tablist

#### What is the current behavior?
aria-expanded attribute is missing

#### What are the steps to reproduce?
https://material.angular.io/components/tabs/overview

#### What is the use-case or motivation for changing an existing behavior?
We are migrating the application to angular and would like maintain the support for a11y

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular : 6.0 and below
Material: 6.0 and below

**Comment by crisbeto** [comment]
According to the [accessibility guidelines for tabs](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel), only `aria-selected` is required on the `tab`, which we already have.

**Comment by VamsiDharmaji** [comment]
@crisbeto , I was referring to role="tablist"  ( If aria-multiselectable is null or set to "false" **within Tablist** ),  element with role="tab" is ARIA compliant

**Comment by josephperrott** [comment]
From the [w3's aria docs for tab](https://www.w3.org/TR/wai-aria-1.1/#tab):

> For a single-selectable `tablist`, authors **SHOULD** hide other `tabpanel` elements from the user until the user selects the tab associated with that tabpanel. For a multi-selectable `tablist`, authors **SHOULD** ensure each visible `tabpanel` has its `aria-expanded` attribute set to `true`, and that the remaining hidden `tabpanel` elements have their `aria-expanded` attributes set to `false`.

Based on this documentation, we should not be including `aria-expanded` as `MatTab`s only allow a single-selectable tab list.  Since we do not support multi-selectable tab lists, I am going to close this issue out.
